database/src/main/scala/cromwell/database/liquibase/DiffResultFilter.scala

```diff
@@ -159,5 +171,7 @@ object DiffResultFilter {
       * @return The updated diff result.
       */
     def filterChangedObjects(filters: Seq[DiffFilter]) = filter(diffResult, filters, (_, _) => false)
+
+    def filterOldeObjects = filter(diffResult, Seq.empty, isOldeObject)
```

Comment:
if we delete the old tables... are we going to transfer the data into the
new structure?  should that be a new ticket?

-------------------------------
Kristian Cibulskis
Chief Architect, Data Sciences & Data Engineering
Broad Institute of MIT and Harvard
kcibul@broadinstitute.org



On Mon, Jun 6, 2016 at 4:02 PM, mcovarr <notifications@github.com> wrote:

> In
> database/src/main/scala/cromwell/database/liquibase/DiffResultFilter.scala
> <https://github.com/broadinstitute/cromwell/pull/957#discussion_r65958109>
> :
>
> > @@ -159,5 +171,7 @@ object DiffResultFilter {
> >        * @return The updated diff result.
> >        */
> >      def filterChangedObjects(filters: Seq[DiffFilter]) = filter(diffResult, filters, (_, _) => false)
> > +
> > +    def filterOldeObjects = filter(diffResult, Seq.empty, isOldeObject)
>
> I will TODO PBE this as something that should be deleted after the
> offending Olde tables are deleted.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/broadinstitute/cromwell/pull/957/files/d81508d6f1afc11a0bf3656200c215f39fb87a67#r65958109>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABW4g3W0h5-CRard0QEz7O9p3RKXNET4ks5qJHzYgaJpZM4IvN3Q>
> .
>